test_test.go

```diff
@@ -215,4 +215,4 @@ func testInnerTests(t *testing.T) {
 	if test.Tests[2].Error == nil || !strings.Contains(test.Tests[2].Error.Error(), "status") {
 		t.Errorf("expected must status to fail")
 	}
-}
+}
```

Comment:
I prefer newline at the end of the files, but feel free to leave this or change it.